FhirToDataLake/test/Microsoft.Health.AnalyticsConnector.Core.UnitTests/Jobs/FhirToDatalakeSchedulerServiceTests.cs

```diff
@@ -619,7 +619,7 @@ public async Task GivenNewJob_WhenRunAsync_ThenTheInitialTriggerEntityShouldBeCr
             CurrentTriggerEntity currentTriggerEntity = await metadataStore.GetCurrentTriggerEntityAsync((byte)QueueType.FhirToDataLake, CancellationToken.None);
             Assert.NotNull(currentTriggerEntity);
 
-            Assert.Equal(1, currentTriggerEntity.OrchestratorJobId);
+            Assert.Equal(0, currentTriggerEntity.OrchestratorJobId);
```

Comment:
Why the value was 1 previously?

Reply:
The start jobId for orchestrator job is set to 1 in the tests previously.